app/Repositories/Three/PostRepository.php

```diff
@@ -83,8 +83,8 @@ public function create(array $data, $signupId, $authenticatedUserRole = null)
             'northstar_id' => $signup->northstar_id,
             'campaign_id' => $signup->campaign_id,
             'quantity' => isset($data['quantity']) ? $data['quantity'] : null,
-            'type' => isset($data['type']) ? $data['type'] : 'photo',
-            'action' => isset($data['action']) ? $data['action'] : null,
+            'type' => $data['type'],
+            'action' => $data['action'],
             'url' => $fileUrl,
```

Comment:
I removed the `isset` check because in `v3`, these are required.

Reply:
I think we should keep these because this repository could be used in other places other than `v3` routes and we want to allow them to be flexible.

Reply:
ah ok that makes sense! Changed back :)